index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
+CDDL definitions.
+
+<!-- TODO: Add a capability that indicates support for the
+remote playback protocol.
+See https://github.com/webscreens/openscreenprotocol/issues/123 -->
+<!-- TODO: Frequency to update current-time (HtmlMediaElement does every 250ms?) -->
+<!-- TODO: extended mime types in source tag? -->
+<!-- TODO: media queries replacing URLs? -->
+
+To learn which receivers are [=compatible remote playback device=]s (also called
+available [=remote playback devices=]) for a particular URL or set of URLs, the
+controller may send a remote-playback-availability-request message with the
+following values:
+
+: urls
+:: A list of [=media resources=].  Must not be empty.
+
+: headers 
+:: headers that the receiver should use to fetch the
+    urls.  For example,
+    [[REMOTE-PLAYBACK#establishing-a-connection-with-a-remote-playback-device|section 6.2.4 of
+    the Remote Playback API]] says that the Accept-Language header should be
+    provided.
+
+: watch-duration
+:: The period of time that the controller is interested in receiving updates
+    about the URLs, should the availability change.
+
+: watch-id
+:: An identifier the receiver may use when sending updates about URL
+    availability so that the controller knows which URLs the receiver is referring
+    to.
+
+In response, the receiver should send a remote-playback-availability-response
+message with the following values:
+  
+: url-availabilities
+:: A list of URL availability states (available,
+    unavailable, or invalid).  Each state must correspond to the matching URL
+    from the request by list index.
+
+
+The receivers should later (up to the current time plus request
+watch-duration) send remote-playback-availability-event  messages if
+URL availabilities change.  Such events contain the following values:
+
+: watch-id
+:: The watch-id given in the presentation-url-availability-response,
+    used to refer to the presentation URLs whose availability has changed.
+
+: url-availabilities
+:: A list of URL availability states (available,
+    unavailable, or invalid).  Each state must correspond to the URLs from the
+    request referred to by the watch-id.
+
+Note that these messages are not broadcasted to all controllers. They are sent
+individually to controllers that have requested availability for the URLs that
+have changed in availability state within the watch duration of the original
+availability request.
+
+To save power, the controller may disconnect the QUIC connection and
+later reconnect to send availablity requests and receive availability
+responses and updates.
+
+
+To start remote playback, the controller may send a
+remote-playback-start-request message to the receiver with the following
+values:
+
+: remote-playback-id 
+:: An identifier that uniquely identifies the remote playback from the
+    controller to the receiver.  It does not need to be unique across all remote
+    playbacks from all controllers to that receive nor unique across all remote
```

Comment:
The receiver can already do that (keep a table of [controller, remote-playback-id]) with this rule because the receiver will see unique IDs from that controller (it's unique for the [controller, receiver] pair).  I don't think having it unique from one controller to all receivers buys us anything.

However, we can consider allowing the receiver to choose the ID rather than the controller, like we do for presentations, which would simplify things (because the receiver can just create a unique ID across all controllers).  I'll make a github issue to track.